pkg/os/util_linux.go

```diff
@@ -20,3 +35,23 @@ func WriteToFileAsRoot(reason, content, filepath string) error {
 	}
 	return nil
 }
+
+func GetReadablePath(paths []string) (string, error) {
```

Comment:
`GetFirstExistentPath` or something, but the name `GetReadablePath ` makes not much sense to me. I would like to see a naming that resembles the fact that the first occurence is used when it exists.